src/microbe_stage/MicrobeHUD.cs

```diff
@@ -838,4 +838,25 @@ private void OnTranslationsChanged()
         UpdateColonySizeForMulticellular();
         UpdateColonySizeForMacroscopic();
     }
+
+    private void ToggleProcessPressed(ChemicalEquation equation)
+    {
+        if (!stage!.HasAlivePlayer || !stage!.Player.Has<BioProcesses>())
+            return;
+
+        ref var processes = ref stage.Player.Get<BioProcesses>();
+
+        if (processes.ActiveProcesses == null)
+            return;
+
+        for (int i = 0; i < processes.ActiveProcesses.Count; ++i)
```

Comment:
For more efficient code processes.ActiveProcesses and the Count should be stored in local variables to remove the need to load the variables multiple times.

Reply:
Count is being get only one time right?

Reply:
`for (... ; this part here; ...)`

the "this part here" executes once for each loop *iteration*. So Count property causes a method call each iteration of the loop. Whereas fetching it once before the loop and storing in a local variable causes C# just to load the local back for comparison which is much simpler operation.